### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by maaloum**
In this project, we :

- Create new branch : contact-form branch.
- Add linters 
- Add Index.html and style.css
- Used Flexbox
- Linked Formspree.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -143,6 +141,22 @@ <h2>About <br> Myself</h2>
...
                 </div>
         </section>
+        <section class="contact-form">
+            <div class="first-container">
+                <h2>Contact me </h2>
+                <p>If you have an application you <br> are interested in developing, <br> a feature that you need built or a <br> project that needs coding. <br> I’d love to help with it </p>
+            </div>
+            <form action="https://formspree.io/f/mzboldye" method="post">
+                <ul>
+                    <li><input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Yeremias"  required></li>
```
Comment (ShahierNashaat) [inline index.html:151]: - [ ] Kindly write in the placeholder a sentence that helps the user to know what to write like `Your name` but `Yeremias` makes the user confused about what to write.

File: index.html [hunk 2]
```diff
@@ -143,6 +141,22 @@ <h2>About <br> Myself</h2>
...
+                    <li><input type="email" id="email" name="user_email" placeholder="natajayanj14@gmail.com" required></li>
```
Comment (ShahierNashaat) [inline index.html:152]: - [ ] Kindly write in the placeholder a sentence that helps the user to know what to write like `Your email` but `natajayanj14@gmail.com` makes the user confused about what to write.

File: index.html [hunk 3]
```diff
@@ -143,6 +141,22 @@ <h2>About <br> Myself</h2>
...
+                    <li><textarea name="message" id="msg" cols="30" rows="10" maxlength = "500" placeholder="Write your message here"></textarea></li>
```
Comment (ShahierNashaat) [inline index.html:153]: - [ ] Kindly be noticed that you should add the `required` attribute to the textarea as it has to be entered when the user submits.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @maaloum 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**